viseron/webserver/__init__.py

```diff
@@ -106,8 +122,23 @@ def create_application():
                 (r"/ws-stream", RegularSocketHandler),
                 (r"/websocket", WebSocketHandler),
                 (r"/(?P<camera>[A-Za-z0-9_]+)/stream", DeprecatedStreamHandler),
+                (r"/ui/", IndexHandler),
+                (r"/ui/about", AboutHandler),
+                (r"/ui/cameras", CamerasHandler),
+                (r"/ui/index", IndexHandler),
+                (r"/ui/recordings", RecordingsHandler),
+                (r"/ui/settings", SettingsHandler),
+                (
+                    r"/recordings/(.*)",
+                    tornado.web.StaticFileHandler,
+                    {"path": RECORDER_PATH},
```

Comment:
This may not be secure and allow directory traversal. Haven't tested that yet.

Reply:
Think its fine to implement it like this as a start